src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java

```diff
@@ -40,6 +40,7 @@ protected String callbackKey(CallbackVersion version, CallbackType type, String
         return String.format("%s%s%s", formattedVersion, type.getValue(), formattedPageId);
     }
 
+    @SuppressWarnings("java:S1172")
     public String camundaActivityId(CallbackParams callbackParams) {
```

Comment:
in this case, maybe the warning is correct as callbackParams doesnt seem to be used.

Reply:
I think this methods are intended to be overridden which is why they the args are not used here